bzr.go

```diff
@@ -64,6 +65,12 @@ func (s BzrRepo) Vcs() Type {
 
 // Get is used to perform an initial clone of a repository.
 func (s *BzrRepo) Get() error {
+
+	basePath := path.Join(s.LocalPath(), "..")
```

Comment:
Actually, instead of using `Join` an alternative approach should be used.
```go
basePath := filepath.Dir(filepath.FromSlash(s.LocalPath()))
```

This will do 2 things:

1. It will convert the directory separator into the correct one for the OS.
2. It will let the `filepath` package figure out the parent directory. That's going to be far more well tested than doing it ourselves.